src/ServiceBus.AttachmentPlugin.Tests/ServiceBus.AttachmentPlugin.Tests.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="MarkdownSnippets" Version="16.5.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
```

Comment:
u should enable dependabot

Reply:
I have it turned on. Don't like that it sends updates one by one.